### DISCUSSION TITLE
Small-scale dissipation

### DISCUSSION THREAD

**Discussion by mtbrolly** [FourierFlows/GeophysicalFlows.jl#280] [Q&A] [answered]
Hi everyone,

I've just started playing with GeophysicalFlows.jl and have quite a naive question.

It looks like there is no explicit small-scale dissipation specified in the 2D decaying example; n\nu = 1, but \nu = 0. With zero viscosity I would expect the simulation to become unstable, but that isn't the case. Is there some magic going on in FourierFlows.jl that I should know about?

Thanks in advance.

**Comment by mtbrolly** [comment, marked as answer]
I've realised there is an exponential filter applied via `stepper="FilteredRK4"`. I think it deserves a comment in the documentation.

**Reply by glwagner** [reply]
I agree! A comment here: https://github.com/FourierFlows/GeophysicalFlows.jl/blob/e25b664d20c6cc2a902b011fa54107d4a6b603d0/examples/twodnavierstokes_decaying.jl#L45-L48 would be useful.

**Reply by navidcy** [reply]
It's written in some other [examples](https://fourierflows.github.io/GeophysicalFlowsDocumentation/stable/literated/singlelayerqg_betadecay/#Problem-setup). I'll open a PR to add a remark like this in all of them.
